index.html

```diff
@@ -224,5 +224,19 @@ <h3>Skills</h3>
             </div>
           </section>
             </main>
-</body>
-</html>
\ No newline at end of file
+            <footer>
+              <h2 class="contact-me">Contact me</h2>
+              <p class="footer-p">
+                If you have an application you are interested in developing,
+                 a feature that you need built or a project that needs coding.
+                  I’d love to help with it 
+              </p>
+              <form  class="feedback" action="https://formspree.io/html/" method="post">
```

Comment:
- [ ] Please note that the action link https://formspree.io/html/ is not a valid formspree URL, you have to create an account at [formspree ](https://formspree.io/register)and then generate your own id and then you need to add it to your form action, the correct link format should be like this: _**`https://formspree.io/f/{form_id}`**_ , where the {form_id} is the key generated in your formspree account , kindly fix that 🙏 

![image](https://user-images.githubusercontent.com/89282221/200800061-94e5dd6e-0a19-423b-bb86-8d68dabcadcd.png)